go/builder.go

```diff
@@ -540,6 +545,24 @@ func (b *Builder) Slot(slotnum int) {
 	b.vtable[slotnum] = UOffsetT(b.Offset())
 }
 
+// FinishWithFileIdentifier finalizes a buffer, pointing to the given `rootTable`.
+// as well as applys a file identifier
+func (b *Builder) FinishWithFileIdentifier(rootTable UOffsetT, fid []byte) {
+	if fid == nil || len(fid) != fileIdentifierLength {
+		panic(fmt.Sprintf(
```

Comment:
If you made this "incorrect file identifier length" you can drop the `fmt` dependency (and who knows how much code).

Reply:
updated